### ISSUE TITLE
[qnxlab] Duplicate captions

### ISSUE THREAD

**Issue by bltzkrg22** [ztmir/ztmir-texfactory#53]
Second occurence of the caption _Brak synchronizacji pracy wątków_ is a leftover copy.

In qnxlab6.tex, search for:
`\lstinputlisting[caption=Brak synchronizacji pracy wątków,style=MyCStyle,label=src:synchrobarrier]{src/lab6/synchrobarrier.c}`
and replace with:
`\lstinputlisting[caption=Synchronizacja przy pomocy bariery,style=MyCStyle,label=src:synchrobarrier]{src/lab6/synchrobarrier.c}`
or equivalent.

Conversely, the caption _Idea działania bariery_ is unfortunate in an example where no synchronization is used:

``` \begin{figure}[!h]
\centering
\includegraphics[width=0.35\textwidth]{img/thrd_async_example}
\caption{Idea działania bariery}
\label{fig:thrd_async_example}
\end{figure}
```

**Comment by ptomulik** [comment]
Yup, thanks! Apparently we need more sleep :). I've made appropriate patch and will apply it soon.

**Comment by ptomulik** [comment]
I believe this is fixed in #54, so closing for now.
